common/src/main/java/org/opensearch/sdk/SdkClient.java

```diff
@@ -19,18 +19,14 @@
 
 import static org.opensearch.sdk.SdkClientUtils.unwrapAndConvertToException;
 
-public class SdkClient implements SettingsChangeListener {
+public class SdkClient {
```

Comment:
Do we really need to change this?

I understand if we make the settings static, there's no need for the listener. But even if we keep the settings static this piece of code shouldn't break the functionality. 

IMO, it's even a two way door, like if we want to make the settings dynamic, corresponding listener solution is already in place. On top of that we haven't finalized the nature of this settings yet.